client/src/components/App.js

```diff
@@ -21,12 +22,23 @@ const App = (props) => {
         setCurrentUser(null)
       })
   }, [])
+  
   return (
     <Router>
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
-          <h2>Hello from Trails</h2>
+        <div>
+          <div className="bg-img">
+          <div className="bg-container">
+            <div className="bg-text">
+              <h1>HIKEROO</h1>
+              <h2>All the best trails in Massachusetts</h2>
+              <h3>Let's Go Outside!</h3>
```

Comment:
Do you want this text on all pages below your TopBar?

Reply:
NOT SURE. Would love more input! Do you mean pages OR do you mean Children?

Reply:
This should remain as is, per team.